File: 1-TwoSum/1-TwoSum.java
// Last updated: 3/26/2025, 7:21:32 PM
class Solution {
    public int[] twoSum(int[] nums, int target) {
    for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    return new int[]{i, j};
                }
            }
        }
        return new int[0];
    }
}

File: 1-TwoSum/1-TwoSum.py
# Last updated: 6/5/2025, 2:10:50 PM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        Map = {}

        for i,j in enumerate(nums) :
            difference = target - j
            if difference in Map:
                return [Map[difference],i]
            Map[j] = i    